packages/core/src/components/forms/_form-group.scss

```diff
@@ -86,6 +119,12 @@ Styleguide pt-form-group
       margin: 0 $pt-grid-size 0 0;
       line-height: $pt-input-height;
     }
+
+    // not ideal but baseline doesn't align text perfectly
+    // so we have to align it to the label manually
+    .pt-control {
+      margin-top: $pt-grid-size * 1.2;
```

Comment:
❌ absolutely not. try removing `.pt-large` from the example and it breaks...
![image](https://user-images.githubusercontent.com/464822/28156839-01e64402-67bd-11e7-8617-de517e3831b0.png)

gonna have to do this in a smarter way. perhaps make the height equal to `$input-height[-large]` (same as label)? will take some work to get this right...